bundles/action/pom.xml

```diff
@@ -32,6 +32,7 @@
     <module>org.openhab.action.astro</module>
     <module>org.openhab.action.tinkerforge</module>
     <module>org.openhab.action.harmonyhub</module>
+    <module>org.openhab.action.mqtt</module>
   </modules>
 
-</project>
+</project>
```

Comment:
I'm just curious why this new line is needed? Of course have no issue with this.